package.json

```diff
@@ -7,49 +7,46 @@
     "url": "git+https://github.com/fecgov/openFEC.git"
   },
   "scripts": {
-    "build": "export NODE_OPTIONS=--openssl-legacy-provider && npm run format:lint && webpack --config webpack.production.config.js --colors",
-    "develop": "export NODE_OPTIONS=--openssl-legacy-provider && webpack-dev-server --config webpack.config.js --mode development",
+    "build": "npm run lint && webpack --config webpack.production.config.js",
+    "develop": "webpack-dev-server --config webpack.config.js",
     "format": "prettier --write 'src/**/*.js'",
     "lint": "eslint ./src/**/*.js",
     "format:lint": "npm run format && npm run lint"
   },
   "dependencies": {
-    "@braintree/sanitize-url": "^6.0.0",
-    "immutable": "^3.8.2",
```

Comment:
immutable was added here to upgrade it. at the time it was used by another package. but now it is not needed at all so we can remove it here. same for braintree.